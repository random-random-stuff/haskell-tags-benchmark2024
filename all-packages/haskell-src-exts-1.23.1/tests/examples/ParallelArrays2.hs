-- Should fail without extensions.
dotp_double :: [:Double:] -> [:Double:] -> Double
dotp_double xs ys = sumP [:x * y | x <- xs | y <- ys:]

