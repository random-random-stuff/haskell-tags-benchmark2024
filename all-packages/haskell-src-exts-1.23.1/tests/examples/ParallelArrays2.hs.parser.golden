ParseFailed
  (SrcLoc "tests/examples/ParallelArrays2.hs" 2 17) "Parse error: :"
